mono/mini/mini-arm.h

```diff
@@ -329,7 +329,10 @@ typedef struct MonoCompileArch {
 #define MONO_ARCH_DYN_CALL_SUPPORTED 1
 #define MONO_ARCH_DYN_CALL_PARAM_AREA (DYN_CALL_STACK_ARGS * sizeof (mgreg_t))
 
+#if !(defined(TARGET_ANDROID) && defined(MONO_CROSS_COMPILE))
 #define MONO_ARCH_SOFT_DEBUG_SUPPORTED 1
+#define MONO_ARCH_HAVE_INIT_LMF_EXT 1
+#endif
```

Comment:
because https://github.com/mono/mono/blob/21d5c9c346113d47455d96e545410733bfccf1fb/mono/mini/mini.h#L2801

so, in hindsight, this is probably wrong: the prototype for `mono_arch_init_lmf_ext` should be guarded by `MONO_ARCH_HAVE_INIT_LMF_EXT` and not `MONO_ARCH_SOFT_DEBUG_SUPPORTED`, correct?